content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -22,6 +22,34 @@ Most of the blog's content is about things happening in the core project, but we
 
 Anyone can write a blog post and submit it for review.
 
+### Submit a Post
+
+Blog posts should not be commercial in nature and should consist of content that will apply broadly to the Kubernetes community.
```

Comment:
Blog posts should not be commercial in nature and should consist of original content that applies broadly to the Kubernetes community.

Reply:
Done